.github/workflows/cla.yml

```diff
@@ -0,0 +1,30 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+    paths:
+    - 'publicexamples/**'
```

Comment:
Can we change the path to "public_examples"? It makes it easer to read with the underscore.

Reply:
the folder will be renamed to "public_examples". sounds good ?